planemo_ext/galaxy/tools/linters/inputs.py

```diff
@@ -23,10 +25,40 @@ def lint_inputs(tool_xml, lint_ctx):
             if "format" not in param_attrib:
                 lint_ctx.warn("Param input [%s] with no format specified - 'data' format will be assumed.", param_name)
         # TODO: Validate type, much more...
+
+    conditional_selects = tool_xml.findall("./inputs//conditional")
+    for conditional in conditional_selects:
+        select = conditional.find('./param[@type="select"]')
+        # Should conditionals ever not have a select?
+        if not len(select):
+            lint_ctx.info("Conditional without <param type=\"select\" />")
+            continue
+
+        select_options = select.findall('./option[@value]')
+        select_option_ids = [option.attrib['value'] for option in select_options]
+        whens = conditional.findall('./when')
+        when_ids = [when.attrib['value'] for when in whens]
```

Comment:
Maybe here it would be good to call `lint_ctx.error()` if the `<when>` has no `value` attribute.